### PR TITLE
various UI improvements

### PR DESCRIPTION
**Pull request by pgigis**
### Description of PR
Resolves the following issues of https://github.com/FORTH-ICS-INSPIRE/artemis/issues/56

* Change Hijack AS to Hijacker AS
* Fix indentation on hijacks table
* Make more clear the buttons of navbar
* Change Timewindow phrase on BGP Updates, Hijacks tables and fix bug with custom window.

What component(s) does this PR affect?

- [ ] Back-End (Database, Microservices, Containers, etc)
- [X] Front-End (Flask, API, etc)
- [ ] Docs
- [ ] Build System

Does the PR require changes on other components? If yes, please mark the components:

- [ ] Back-End (Database, Microservices, Containers, etc)
- [ ] Front-End (Flask, API, etc)
- [ ] Docs
- [ ] Build System

### Type

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Docs update
- [ ] None of the above

### Checklist:

- [x] I have read the **[contributing guide](https://github.com/FORTH-ICS-INSPIRE/artemis/blob/master/CONTRIBUTING.md)** and my code conforms to the guidelines.
- [ ] This change requires a change in the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: frontend/webapp/views/actions/actions_view.py [hunk 1]
```diff
@@ -130,6 +130,9 @@ def remove_admin():
     if form.select_field.data is not None:
         user = app.security.datastore.find_user(
             id=form.select_field.data)
+        # Protect admin (user id == 1)
+        if user.id == 1:
+            return redirect('admin/user_management')
```
Comment (vkotronis) [inline frontend/webapp/views/actions/actions_view.py]: returns without a prompt?

Comment (pgigis) [reply frontend/webapp/views/actions/actions_view.py]: Nope. Its for extra security. Manually this action can't be triggered

File: frontend/webapp/static/js/custom/display_info-prod.js [hunk 2]
```diff
@@ -1,14 +1,10 @@
-var $jscomp=$jscomp||{};$jscomp.scope={};$jscomp.ASSUME_ES5=!1;$jscomp.ASSUME_NO_NATIVE_MAP=!1;$jscomp.ASSUME_NO_NATIVE_SET=!1;$jscomp.SIMPLE_FROUND_POLYFILL=!1;$jscomp.defineProperty=$jscomp.ASSUME_ES5||"function"==typeof Object.defineProperties?Object.defineProperty:function(a,c,b){a!=Array.prototype&&a!=Object.prototype&&(a[c]=b.value)};$jscomp.getGlobal=function(a){return"undefined"!=typeof window&&window===a?a:"undefined"!=typeof global&&null!=global?global:a};$jscomp.global=$jscomp.getGlobal(this);
```
Comment (slowr) [inline frontend/webapp/static/js/custom/display_info-prod.js]: You can remove the whole -prod file now.

File: CHANGELOG.md [hunk 3]
```diff
@@ -19,6 +19,11 @@
 ### Fixed
 - Misc code quality improvements
 - Fetch API support for older browsers
+- Change Hijack AS to Hijacker AS
```
Comment (slowr) [inline CHANGELOG.md]: Maybe just say UI fixes (Hijack->Hijacker) etc?

**Pull request merged**